features/support/setup.rb

```diff
@@ -55,18 +70,34 @@ def scenario_name(scenario)
       screen_dir = './screenshots'
     end
     Dir::mkdir(screen_dir) if not File.directory?(screen_dir)
-    sname = scenario_name(scenario)
-    screenshot = File.join(screen_dir, "FAILED_#{sname.gsub(' ','_').gsub(/[^0-9A-Za-z_]/, '')}.png")
-    screenshot_embed_filename = "./screenshots/FAILED_#{sname.gsub(' ','_').gsub(/[^0-9A-Za-z_]/, '')}.png"
-    page.driver.render(screenshot, :full => true)
+    screenshot = File.join(screen_dir, "FAILED_#{scenario.name.gsub(' ','_').gsub(/[^0-9A-Za-z_]/, '')}.png")
+    screenshot_embed_filename = "./screenshots/FAILED_#{scenario.name.gsub(' ','_').gsub(/[^0-9A-Za-z_]/, '')}.png"
+    page.save_screenshot(screenshot, full: true)
     embed screenshot_embed_filename, 'image/png'
+    puts 'Browser console output:'
+    puts '---'
+    for logitem in page.driver.browser.manage.logs.get(:browser)
+      puts logitem
+    end
+    puts '---'
   end
   Capybara.reset_sessions!
 end
 
 Before do |scenario|
   @params = {}
-  Syslog.log(Syslog::LOG_INFO, "Running '#{scenario_name(scenario)}'")
+  case scenario.source.last
```

Comment:
Is this "@scenario_name" used anywhere?

It is mostly refered as scenario.name

Reply:
Let's try and remove it. :+1: